src/app/features/home/components/advertisement/rent-card/rent-card.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit } from '@angular/core';
+import { Observable } from "rxjs";
+import { RentCardService, ICard } from "../../../services/rent-card.service";
+
+
+@Component({
+  selector: 'app-rent-card',
+  templateUrl: './rent-card.component.html',
+  styleUrls: ['./rent-card.component.scss']
+})
+export class RentCardComponent implements OnInit {
```

Comment:
This component should display only one card

In your case, you should have 2 components one for the list of cards and for the card item itself